modules/managed-cluster-notification-severity-levels.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-notifications.adoc
+// * osd_cluster_admin/osd-cluster-notifications.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="managed-cluster-notification-severity_{context}"]
+= Cluster notification severity levels
+
+Each cluster notification has an associated severity level to help you identify notifications with the greatest impact to your business. You can filter cluster notifications according to these severity levels in Hybrid Cloud Console, in the **Cluster history** tab for your cluster.
+
+Red Hat uses the following severity levels for cluster notifications, from most to least severe:
+
+Critical:: Immediate action is required. One or more key functions of a service or cluster is not working, or will stop working soon. A critical alert is important enough to page on-call staff and interrupt regular workflows.
```

Comment:
It looks like those severity levels is not correct according to the API definition https://api.openshift.com/?urls.primaryName=Service%20logs#/
Supported levels includes: Debug, Info, Warning, Error, Fatal, Major, Critical

Reply:
@rawsyntax Can you confirm which severity levels I should be using? I used https://gitlab.cee.redhat.com/service/ocm-service-log#severity-levels as the basis for these definitions but tweaked them to be a little more action-oriented for customers.

Reply:
@laubai Hi,  I could see some of the severity levels mentioned in the doc are missing when compared with UI.
See the severity filter options from UI side.
![image](https://github.com/openshift/openshift-docs/assets/93653094/60bd8b2b-94bb-46d4-83c1-693ca833241c)